website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -82,6 +82,16 @@ Before upgrading to Nomad 1.10, perform the following tasks:
 Refer to [Migrating to Using Workload Identity with Vault][] for more
 details.
 
+#### Consul template implicit workload identity removal
+Nomad will no longer create an implicit Consul identity for workloads that don't
+register services with Consul. Jobs that require Consul tokens for template
+rendering will be required to provide an identity.
+
+Tasks that include template blocks utilizing Consul API functionality such as
+`key`, `node`, and `service` will now need to specify an [`identity`][] block,
```

Comment:
We should still be creating the identity block implicitly when there's a `consul` block present.